src/main/java/com/jcabi/github/Commits.java

```diff
@@ -70,4 +70,12 @@ Commit create(
      */
     @NotNull(message = "commit is never NULL")
     Commit get(@NotNull(message = "sha can't be null") String sha);
+
+    /**
+     * Return a Statuses object for a given SHA
+     * @param sha The sha of the Commit
+     * @return Status
+     */
+    @NotNull(message = "status is never NULL")
+    Statuses statuses(@NotNull(message = "sha can't be NULL") final String sha);
```

Comment:
@zygm0nt this parameter can be not only SHA by also branch or tag name. Please, name is accordingly.